Software/src/LedDeviceDnrgb.cpp

```diff
@@ -0,0 +1,181 @@
+/*
+ * LedDeviceDnrgb.cpp
+ *
+ *	Created on: 06.09.2011
+ *		Author: Mike Shatohin (brunql), Timur Sattarov
+ *		Project: Lightpack
+ *
+ *	Lightpack is very simple implementation of the backlight for a laptop
+ *
+ *	Copyright (c) 2011 Mike Shatohin, mikeshatohin [at] gmail.com
+ *
+ *	Lightpack is free software: you can redistribute it and/or modify
+ *	it under the terms of the GNU General Public License as published by
+ *	the Free Software Foundation, either version 2 of the License, or
+ *	(at your option) any later version.
+ *
+ *	Lightpack is distributed in the hope that it will be useful,
+ *	but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	See the
+ *	GNU General Public License for more details.
+ *
+ *	You should have received a copy of the GNU General Public License
+ *	along with this program.	If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "LedDeviceDnrgb.hpp"
+#include "enums.hpp"
+
+LedDeviceDnrgb::LedDeviceDnrgb(const QString& address, const QString& port, QObject * parent) : AbstractLedDeviceUdp(address, port, parent)
+{
+}
+
+const QString LedDeviceDnrgb::name() const
+{ 
+	return "dnrgb"; 
+}
+
+int LedDeviceDnrgb::maxLedsCount()
+{
+	return 1500;
```

Comment:
these would be better with `MaximumNumberOfLeds::xxx`

Reply:
Done. I'll leave the Ada/Ardu devices alone for now